.github/workflows/linux_conditional_compilation.yml

```diff
@@ -212,6 +212,7 @@ jobs:
             tar -czvf ${BUILD_DIR}/openvino_tests.tar.gz \
               tests/ov_cpu_func_tests \
               tests/libopenvino_template_extension.so \
+              tests/libze_loader.so* \
```

Comment:
let's do the same for Windows CC pipeline

Reply:
Added in https://github.com/XinWangIntel/openvino/blob/npu_plugin_snapshot/.github/workflows/windows_conditional_compilation.yml#L236